### PR TITLE
units, wtf: add pt_BR translation

### PR DESCRIPTION
**Pull request by barbaraaliverti**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.pt_BR/linux/units.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# units
+
+> Realiza a conversão entre duas unidades de medida.
+> Digitar 'search{{text}}' no console retornará uma lista de todas as unidades que contêm {{text}}.
```
Comment (navarroaxel) [inline pages.pt_BR/linux/units.md:4]: The `'` is a bug in the English page, and you can add the `More information` link:
```suggestion
> Digitar `search{{text}}` no console retornará uma lista de todas as unidades que contêm `{{text}}`.
> Mais informações: <https://www.gnu.org/software/units/units.html>.
```
You can fix the English page in this PR too.

File: pages.pt_BR/linux/wtf.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# wtf
+
+> Mostra a expansão de acrônimos.
+> Mais informações (em inglês): <https://manpages.debian.org/bsdgames/wtf.6.en.html>.
```
Comment (navarroaxel) [inline pages.pt_BR/linux/wtf.md:4]: We don't mention the link language:
```suggestion
> Mais informações: <https://manpages.debian.org/bsdgames/wtf.6.en.html>.
```

File: pages.pt_BR/linux/wtf.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+- Expandir um acrônimo:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/wtf.md:6]: You shouldn't use infinite, you should use indicative mood, I think you should use:
```suggestion
- Expanda um acrônimo:
```
I could be wrong with this conjugation :grin:

File: pages.pt_BR/linux/wtf.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+`wtf {{IMO}}`
+
+- Especificar um tipo de busca computacional:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/wtf.md:10]:
```suggestion
- Específica um tipo de busca computacional:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @barbaraaliverti, welcome and thanks for your submission. I left some comments.

I don't speak Portuguese, I speak Spanish :argentina: but I tried to help with the translation :upside_down_face:

**Comment by barbaraaliverti** [comment]
Thanks for your review, @navarroaxel :smile:

File: pages/linux/units.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
 
 > Provide the conversion between two units of measure.
 > Typing 'search {{text}}' in the prompt will display a list of all of the units containing {{text}}.
```
Comment (navarroaxel) [inline pages/linux/units.md:4]:
```suggestion
> Typing `search {{text}}` in the prompt will display a list of all of the units containing `{{text}}`.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @barbaraaliverti, just 1 comment.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**